lib/src/Uncertainty/Algorithm/Simulation/PhysicalSpaceCrossEntropyImportanceSampling.cxx

```diff
@@ -59,6 +59,10 @@ PhysicalSpaceCrossEntropyImportanceSampling::PhysicalSpaceCrossEntropyImportance
     activeParameters_ = activeParameters;
```

Comment:
preferably use the list initialization to set these attributes (like you did for solver_)

Reply:
fixed